assets/app/lib/settings.rb

```diff
@@ -7,7 +7,7 @@ module Settings
     DARK = `window.matchMedia('(prefers-color-scheme: dark)').matches`.freeze
     # http://mkweb.bcgsc.ca/colorblind/ 12 color palette
     # change SETTINGS = (n.times ... in models/user.rb accordingly
-    ROUTE_COLORS = %i[#A40122 #008DF9 #00FCCF #FF5AAF #8400CD #FF6E3A].freeze
+    ROUTE_COLORS = %i[#A40122 #008DF9 #00FCCF #FF5AAF #8400CD #FF6E3A #009F81 #FFC33B].freeze
```

Comment:
Systems can run up to 8 trains. Add more route colors from the colorblind palate.